packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/java/summaryofsteps.md

```diff
@@ -7,3 +7,11 @@ The following graphic shows the steps for the sign-in flow with password and pho
 ![Displays the sequence of steps for the Java SDK sign in with password and phone flow](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-seq-1-java.png)
 
 </div>
+
+The following diagram shows an alternative step with the voice feature enabled in your org:
+
+<div class="common-image-format">
+
+![Displays the the voice-enabled alternative step for the Java SDK sign in with password and phone flow](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-seq-2-java.png)
```

Comment:
This diagram shows SMS instead of Voice code.

Reply:
i know :-( I was being lazy ... and just added voice FF, but user still enters SMS. lol. I don't know if voice actually works - I haven't been able to try it out.
I'll update the diagram and description to select voice and make it somewhat generic

Reply:
Voice does work, I can give you my org credentials offline if you'd like to try it.